python/adbc_driver_bigquery/adbc_driver_bigquery/__init__.py

```diff
@@ -52,6 +52,9 @@ class DatabaseOptions(enum.Enum):
     AUTH_CLIENT_SECRET = "adbc.bigquery.sql.auth.client_secret"
     AUTH_REFRESH_TOKEN = "adbc.bigquery.sql.auth.refresh_token"
 
+    #: Specify the location to use for bigquery connection.
```

Comment:
we don't use this as far as I know so it's not needed. We're not maintain the python driver